packages/gnome_docking_library.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Gnome_docking_library < Package
+class Gnome_docking_library < Autotools
   description 'GUsb is a GObject wrapper for libusb1'
   homepage 'https://www.gnome.org/'
   version '3.28.0-0'
```

Comment:
Update to [3.40.0](https://github.com/GNOME/gdl/archive/refs/tags/GDL_3_40_0.tar.gz)?